arc_solvers/models/qa/multi_choice/qa_multi_choice_max_att.py

```diff
@@ -0,0 +1,231 @@
+from allennlp.modules.matrix_attention import MatrixAttention
+from typing import Dict, Optional, AnyStr, List, Any
+
+import torch
+from allennlp.common import Params
+from allennlp.common.checks import ConfigurationError
+from allennlp.data import Vocabulary
+from allennlp.models.model import Model
+from allennlp.modules import FeedForward, Seq2SeqEncoder, MatrixAttention, SimilarityFunction
+from allennlp.modules import TextFieldEmbedder
+from allennlp.nn import InitializerApplicator
+from allennlp.training.metrics import CategoricalAccuracy
+
+from arc_solvers.common.common_utils import update_params
+from arc_solvers.nn.util import embedd_encode_and_aggregate_list_text_field, embedd_encode_and_aggregate_text_field
+
+
+@Model.register("qa_multi_choice_max_att")
+class QAMultiChoiceMaxAttention(Model):
+    """
+    This ``Model`` implements a simple context encoder baseline that models the interaction between question and choice:
```

Comment:
Nit: format these lines to < 100 characters long, so you can read them without scrolling,